### PR TITLE
Create Eric's Bio

### PR DESCRIPTION
**Pull request by githubteacher**
I want to add my bio to this page. 

Any questions about what I've written?
Is there anything else you might like to know about me?

@DarcyCK

### CODE REVIEW

File: githubteacher-bio.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+###Eric Hollenberry
+
+####Location
```
Comment (oriandaleigh) [inline githubteacher-bio.md:3]: w00t!

**Comment by githubteacher** [comment]
@DarcyCK 

Does this look good to 🚢 :shipit: ?

**Comment by githubteacher** [comment]
👍

**Comment by AwesomeKai** [comment]
👍

**Pull request merged**